.mvn/gradle-enterprise.xml

```diff
@@ -22,7 +22,9 @@
 <gradleEnterprise
         xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
-  <enabled>#{env['GRADLE_ENTERPRISE_ACCESS_KEY']?.trim() > ''}</enabled>
+  <!-- Enable Gradle Enterprise extension when GRADLE_ENTERPRISE_ACCESS_KEY is set and the build isn't
```

Comment:
I would expand the comment after with:
```
GITHUB_HEAD_REF contains the name of the branch.
The regular expression:
- `(?i)` - This flag makes the matching case insensitive.
```

I would rewrite the comment for clarity:
```
<!-- Enable Gradle Enterprise extension when all of the  following conditions match:
1. GRADLE_ENTERPRISE_ACCESS_KEY is set
2. The branch or repository names do not denote work in progress. In effect, it does not contain either "wip," "experiment," or "private."
-->